08_Loading_Elf/02_Elf_Theory.md

```diff
@@ -0,0 +1,117 @@
+# Executable Linker Format
+
+The executable and linker format (ELF) is a multi-purpose format: it can be used to store compiled (but not linked) object files, as well as fully linked programs/dynamic libraries.
+
+The format has four main sections:
+
+- The ELF header. This contains the magic number used to identify it as an ELF, as well as information about the architecture the ELF was compiled for, the target operating system and other useful info.
```

Comment:
maybe use a different font decoration for the name part, like: and replace the . with  a dash or : ? 

- _The ELF header_: This contains

same for the items below.